migrate/20230915_postgres.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:postgres_server) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :updated_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :project_id, :project, type: :uuid, null: false
```

Comment:
Vm doesn't have `project_id`. The association is established via tags. Should we do same thing here? Or make this thing to vm?

For authorizing users to access resources, we need tags anyway.

Reply:
Resolving this now. We can continue this discussion in main postgres PR.